money/tests/test_money.py

```diff
@@ -232,3 +233,29 @@ def test_format(self):
         assert Money('5.56', Currency.EUR).format('en_UK') == '€5.56'
         assert Money('10', Currency.JPY).format() == '¥10'
         assert Money('94', Currency.JPY).format('ja_JP') == '￥94'
+
+    def test_rounding_per_operation(self):
+        assert Money.rounding_per_operation is True
+
+        instance = Money('1.00')
+
+        Money.set_rounding_per_operation(False)
+
+        assert Money.rounding_per_operation is False
+        assert instance.rounding_per_operation is False
+
+        with pytest.raises(TypeError):
+            Money.set_rounding_per_operation('abcd')
+
+    def test_rounding_type(self):
+        assert Money.rounding_type == ROUND_HALF_UP
+
+        instance = Money('1.00')
+
+        Money.set_rounding_type(ROUND_05UP)
+
+        assert Money.rounding_type == ROUND_05UP
+        assert instance.rounding_type == ROUND_05UP
```

Comment:
rather than testing that the flag is set, could you test that mathematical operations are rounded appropriately?